specs/ff-asn-selected-filter-cloud/ff-asn-selected-filter-cloud.md

```diff
@@ -1,86 +1,212 @@
-# ff-asn-selected-filter-cloud
-The `ff-asn-selected-filter-cloud` displays all active filters using `ff-asn-group-element`'s.
+# ff-filter-cloud
```

Comment:
What do you think about `selected-filter-cloud`? we have an element called `ff-tagcloud` in 1.x. I think it's a bit missleading if we just call it `filter-cloud`